Src/BootCamp.Chapter/Lesson3.cs

```diff
@@ -0,0 +1,68 @@
+﻿/*
+ * ********Lesson Notes for me here********
+ * Requirements for this bootcamp lesson(lesson 3)
+ *  1. Need to have refactored code to recieve inputs
+ *  2. Need to gather the following pieces of information:
+ *      -Name(string}
+ *      -Age(int)
+ *      -Height(Float)[This will be in Meters]
+ *      -Weight(Float)[This will be in Killograms]
+*/
+
+using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Globalization;
+
+namespace BootCamp.Chapter
+{
+    public class Lesson3
+    {
+        public static void RunLesson()
+        {
+            string name;
+            int age;
+            float personHeight,
+                  personWeight,
+                  bmi;
+
+            for (int i = 0; i < 3; i++)
+            {
+                name = GetString("What is your name?: ");
+                age = GetInteger("What is your age?: ");
+                personHeight = GetFloat("What is your height?: ");
+                personWeight = GetFloat("What is your weight?: ");
```

Comment:
Usually the getter and setter functions/methods have this form:
`<type> GetX()` and a `void SetX(<type> x)` with exceptions like `<type> GetElement(int index)`.
As in Checks.cs suggested the naming for requested inputs is something like prompt (which you also have in JS for popup request -> inputs).

Reply:
For now keep it this way with the resolved formatting of the variables going along with the inputs until later lessons to keep things simplified?

Reply:
Yeah for me that is fine. @Almantask may have the last word on that one though.